tests/draft2020-12/unevaluatedItems.json

```diff
@@ -485,5 +485,183 @@
                 "valid": false
             }
         ]
+    },
+    {
+        "description": "unevaluatedItems depends on adjacent contains",
+        "schema": {
+            "prefixItems": [true],
+            "contains": {"type": "string"},
+            "unevaluatedItems": false
+        },
+        "tests": [
+            {
+                "description": "second item is evaluated by contains",
+                "data": [ 1, "foo" ],
+                "valid": true
+            },
+            {
+                "description": "contains fails, second item is not evaluated",
+                "data": [ 1, 2 ],
+                "valid": false
+            },
+            {
+                "description": "contains passes, second item is not evaluated",
+                "data": [ 1, 2, "foo" ],
+                "valid": false
+            }
+        ]
+    },
+    {
+        "description": "unevaluatedItems depends on nested contains",
+        "schema": {
+            "anyOf": [
+                {
+                    "prefixItems": [{"type": "integer"}]
+                },
+                {
+                    "contains": {"type": "string"}
+                }
+            ],
+            "unevaluatedItems": false
+        },
+        "tests": [
+            {
+                "description": "single integer is evaluated by prefixItems",
+                "data": [ 1 ],
+                "valid": true
+            },
+            {
+                "description": "single string is evaluated by contains",
+                "data": [ "foo" ],
+                "valid": true
+            },
+            {
+                "description": "second item is evaluated by contains",
+                "data": [ 1, "foo" ],
+                "valid": true
+            },
+            {
+                "description": "second item is not evaluated, contains fails",
+                "data": [ 1, 2 ],
+                "valid": false
+            },
+            {
+                "description": "second item is not evaluated, contains passes",
+                "data": [ 1, 2, "foo" ],
+                "valid": false
+            }
+        ]
+    },
```

Comment:
I think this set of tests can be removed. We don't need to test `prefixItems` again and we aren't testing anything that hasn't been covered elsewhere.

```suggestion
```

Reply:
I'm fine with dropping this as it is covered in the "multiple nested contains" test case.